.github/workflows/commit.yml

```diff
@@ -68,9 +68,11 @@ jobs:
           FERROCENE_CUSTOM_LLVM: /usr/lib/llvm-19
           FERROCENE_HOST: x86_64-unknown-linux-gnu
           SCRIPT: |
-            ./x.py test tidy
-            ./x.py check library compiler/rustc
-            ./x.py run ferrocene/tools/traceability-matrix
+            set -e
+            ./x test tidy
+            ./x check library compiler/rustc
+            ./x run ferrocene/tools/traceability-matrix
+            ./x test collect-license-metadata
```

Comment:
If this check takes more than a few seconds to run we should also probably extend the timeout to like 20min.